manifest_staging/charts/secrets-store-csi-driver/README.md

```diff
@@ -107,3 +107,4 @@ The following table lists the configurable parameters of the csi-secrets-store-p
 | `providerHealthCheckInterval`           | Provider healthcheck interval duration                                                                                                                                         | `2m`                                                    |
 | `imagePullSecrets`                      | One or more secrets to be used when pulling images                                                                                                                             | `""`                                                    |
 | `tokenRequests`                         | Token requests configuration for the csi driver. Refer to [doc](https://kubernetes-csi.github.io/docs/token-requests.html) for more info. Supported only for Kubernetes v1.20+ | `""`                                                    |
+| `namespace`                             | If given, installs the driver in the given namespace                                               | `""`                                             |
```

Comment:
Could you add empty entry in values.yaml?

Reply:
Have added. Thanks.